app/cafe.py

```diff
@@ -0,0 +1,19 @@
+import datetime
+
+from app.errors import (NotVaccinatedError,

```

Comment:
This import statement is not following the recommended style. It is better to use parentheses `( )` while importing multiple classes/modules to improve readability. Please, revise it.